packages/cloud-cognitive/src/components/InlineEdit/_storybook-styles.scss

```
@@ -5,9 +5,9 @@
 // LICENSE file in the root directory of this source tree.
 //
 
-@import './index';
-
 // TODO: add any additional styles used by InlineEdit.stories.js
+@import '../../global/styles/project-settings';
+@import '../../global/styles/mixins';
```

Not sure why you import mixins here -- you don't seem to be using it.

```suggestion
@import '../../global/styles/project-settings';
```